### ISSUE TITLE
Dependency version regressions in v0.2.1

### ISSUE THREAD

**Issue by sedbytes** [SyedAhkam/swim#1]
I added versioning to `swim` crate local dependencies in f480af2efc1fb76cc8f9d28f3e91124a6b67fb8e which introduced a regression in published crates.io version.

The published version is unable to locate newly re-exported types:

```rs
error[E0433]: failed to resolve: could not find `routerify` in `swim_core`
   --> /home/syed/.cargo/registry/src/github.com-1ecc6299db9ec823/swim-0.2.1/src/lib.rs:155:24
    |
155 |     pub use swim_core::routerify::ext::RequestExt;
    |                        ^^^^^^^^^ could not find `routerify` in `swim_core`

error[E0432]: unresolved import `swim_core::async_trait`
   --> /home/syed/.cargo/registry/src/github.com-1ecc6299db9ec823/swim-0.2.1/src/lib.rs:154:13
    |
154 |     pub use swim_core::async_trait;
    |             ^^^^^^^^^^^^^^^^^^^^^^ no `async_trait` in the root

error[E0432]: unresolved import `swim_core::async_trait`
   --> /home/syed/.cargo/registry/src/github.com-1ecc6299db9ec823/swim-0.2.1/src/lib.rs:166:9
    |
166 | pub use swim_core::async_trait;
    |         ^^^^^^^^^^^^^^^^^^^^^^ no `async_trait` in the root
    |
help: consider importing this unresolved item instead
    |
166 | pub use crate::prelude::async_trait;
    |         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~

error[E0432]: unresolved import `swim_core::hyper`
   --> /home/syed/.cargo/registry/src/github.com-1ecc6299db9ec823/swim-0.2.1/src/lib.rs:167:9
    |
167 | pub use swim_core::hyper;
    |         ^^^^^^^^^^^^^^^^ no `hyper` in the root

error[E0432]: unresolved import `swim_core::routerify`
   --> /home/syed/.cargo/registry/src/github.com-1ecc6299db9ec823/swim-0.2.1/src/lib.rs:168:9
    |
168 | pub use swim_core::routerify;
    |         ^^^^^^^^^^^^^^^^^^^^ no `routerify` in the root

Some errors have detailed explanations: E0432, E0433.
For more information about an error, try `rustc --explain E0432`.
error: could not compile `swim` due to 5 previous errors
```

**Comment by sedbytes** [comment]
The solution seems to be directly depending on specific version strings. Including the patch component.

**Comment by sedbytes** [comment]
Fixed by pinning `swim_core` and `swim_util` to `0.2.1`.

v0.2.2 released in a64a320dd3832b9759b4198230e1e1e1e3e90317
